ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-pathfinder1.cfg

```diff
@@ -100,6 +214,31 @@ interface Vxlan1
    vxlan udp-port 4789
    vxlan vrf default vni 1
 !
+application traffic recognition
+   !
+   application ipv4 CONTROL-PLANE-APPLICATION
+      destination prefix field-set CONTROL-PLANE-APP-DEST-PREFIXES
+   !
+   application ipv4 microsoft-teams
```

Comment:
Do we have to define these? Are they not part of the defaults?

Reply:
better tests with comments [168aff6](https://github.com/aristanetworks/ansible-avd/pull/3446/commits/168aff62d02d1f3b461a969dc786ff3fd197e2a5)